firefox-ios/firefox-ios-tests/Tests/XCUITests/A11ySettingsTests.swift

```diff
@@ -0,0 +1,1208 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Testing
+
+class A11ySettingsTests: BaseTestCase {
+    func testSettingMenuPageAudit() throws {
```

Comment:
Nit: testSettingMenuPageAudit() or testSetting**s**MenuPageAudit() with a 's' in settings. It is the same in all test names. Not an issues :) but we usually use settings